pkg/operator/configobservation/apiserver/observe_send_retry_after_while_not_ready_once.go

```diff
@@ -0,0 +1,76 @@
+package apiserver
+
+import (
+	"fmt"
+	"reflect"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const sendRetryAfterWhileNotReadyOnceKey = "send-retry-after-while-not-ready-once"
+
+// ObserveSendRetryAfterWhileNotReadyOnce ensures that send-retry-after-while-not-ready-once is set for SNO clusters.
+func ObserveSendRetryAfterWhileNotReadyOnce(genericListers configobserver.Listers, _ events.Recorder, existingConfig map[string]interface{}) (ret map[string]interface{}, errs []error) {
+	defer func() {
+		// Prune the observed config so that it only contains apiServerArguments field.
+		ret = configobserver.Pruned(ret, []string{"apiServerArguments"})
```

Comment:
can we lower the scope to `send-retry-after-while-not-ready-once` only, similar to  `ObserveShutdownDelayDuration`?